packages/p2p/src/components/modal-manager/modals/error-modal/error-modal.jsx

```diff
@@ -21,6 +21,7 @@ const ErrorModal = ({
             has_close_icon={has_close_icon}
             is_open={is_modal_open}
             title={error_modal_title}
+            toggleModal={onClose ?? hideModal}
```

Comment:
for some reason the error modals in p2p can have the x icon but then we can't press them since it was missing the toggleModal function here so i decided to add it